### PR TITLE
fixed ur_msg dependency

### PR DESCRIPTION
**Pull request by relffok**
Minor dependency fixes, same PR as: https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/336

### CODE REVIEW

**Comment by destogl** [comment]
Thanks for this!

@fmauch should we use `foxy-devel` or `foxy` branch?

**Review by destogl — APPROVED** [review]
All good from my side (except missing new lines at the end of the file).

Also @fmauch should check if we should rather use `foxy` branch.

File: Universal_Robots_ROS2_Gazebo_Simulation-not-released.galactic.repos [hunk 1]
```diff
@@ -13,5 +13,5 @@ repositories:
     version: main
   ur_msgs:
     type: git
-    url: https://github.com/destogl/ur_msgs.git
-    version: ros2
+    url: https://github.com/ros-industrial/ur_msgs.git
+    version: foxy-devel
```
Comment (destogl) [inline Universal_Robots_ROS2_Gazebo_Simulation-not-released.galactic.repos]:
```suggestion
    version: foxy-devel

```

File: Universal_Robots_ROS2_Gazebo_Simulation-not-released.rolling.repos [hunk 2]
```diff
@@ -24,5 +24,5 @@ repositories:
     version: main
   ur_msgs:
     type: git
-    url: https://github.com/destogl/ur_msgs.git
-    version: ros2
+    url: https://github.com/ros-industrial/ur_msgs.git
+    version: foxy-devel
```
Comment (destogl) [inline Universal_Robots_ROS2_Gazebo_Simulation-not-released.rolling.repos]:
```suggestion
    version: foxy-devel

```

File: Universal_Robots_ROS2_Gazebo_Simulation.rolling.repos [hunk 3]
```diff
@@ -39,5 +39,5 @@ repositories:
     version: main
   ur_msgs:
     type: git
-    url: https://github.com/destogl/ur_msgs.git
-    version: ros2
+    url: https://github.com/ros-industrial/ur_msgs.git
+    version: foxy-devel
```
Comment (destogl) [inline Universal_Robots_ROS2_Gazebo_Simulation.rolling.repos]:
```suggestion
    version: foxy-devel

```

**Comment by fmauch** [comment]
Sorry for delaying my answer... For now it does not make any difference, as they are the same. In the future, `foxy` will **always** be in sync with the currently released version so it would make sense to use `foxy-devel` in a situation where you want to test against upstream devel so to say.

However, since this will be removed after the next syncs (for rolling this is [already through](https://discourse.ros.org/t/new-packages-for-ros-2-rolling-ridley-2022-04-22/25213)) I don't really care.

File: Universal_Robots_ROS2_Gazebo_Simulation-not-released.rolling.repos (hunk 2 above)
Comment (fmauch) [inline Universal_Robots_ROS2_Gazebo_Simulation-not-released.rolling.repos:28]: This is no longer necessary, as ur_msgs is released for rolling.

File: Universal_Robots_ROS2_Gazebo_Simulation-not-released.galactic.repos (hunk 1 above)
Comment (fmauch) [inline Universal_Robots_ROS2_Gazebo_Simulation-not-released.galactic.repos:17]: This will become obsolete rather soon when the next sync happens.

**Review by fmauch — COMMENTED** [review]
I would vote for leaving the galactic-not-released entry out of this as this will become obsolete rather soon. I expect the galactic sync being announced any day now.

Therefore, I am neither approving nor denying this PR.

File: Universal_Robots_ROS2_Gazebo_Simulation-not-released.galactic.repos (hunk 1 above)
Comment (destogl) [reply Universal_Robots_ROS2_Gazebo_Simulation-not-released.galactic.repos:17]: yes, let's remove this entry form there fully.

**Comment by fmauch** [comment]
With #20 this probably got obsolete.

**Pull request closed without merging**